Lombiq.HelpfulLibraries.SourceGenerators/Lombiq.HelpfulLibraries.SourceGenerators/Readme.md

```diff
@@ -0,0 +1,60 @@
+# Lombiq.HelpfulLibraries.SourceGenerators
+
+A collection of helpful source generators.
+**When using one of the generators you must run a build before errors will go away**
+
+- [ConstantFromJsonGenerator.cs](ConstantFromJsonGenerator.cs): A source generator that creates a constant from a JSON file.
+
+## Lombiq.HelpfulLibraries.SourceGenerators.Sample
+
+A project that references source generators. It is used to test and showcase the source generators.
+
+## Lombiq.HelpfulLibraries.SourceGenerators.Tests
+
+Unit tests for source generators.
```

Comment:
This is not necessary, you can put the unit tests into `Lombiq.HelpfulLibraries.Tests` as I mentioned above.